lighthouse/classes/messages/warehouse_messages.py

```diff
@@ -0,0 +1,112 @@
+from lighthouse.messages.message import Message
```

Comment:
Have you got a similar setup in your vscode settings to organise imports? When another dev saves this file there will be a diff but no code changes. This is mine:
```
"[python]": {
    "editor.rulers": [120],
    "editor.codeActionsOnSave": {
      "source.organizeImports": true
    },
    "gitlens.codeLens.symbolScopes": ["!Module"],
    "editor.wordBasedSuggestions": false,
    "editor.formatOnSave": true
  },
  ```